### ISSUE TITLE
Docker Images

### ISSUE THREAD

**Issue by norberttech** [flow-php/flow#511]
Create and upload to dockerhub flow-php docker images with phar and latest PHP version inside allowing to load dataframe from single PHP file using flow runtime. 

```shell
docker run -t --rm -v $PWD:/dataframes flow-php/flow ./dataframe.php
```

Add examples to readme how to execute flow dataframes directly from Docker

**Comment by norberttech** [comment]
waiting for dockerhub open source organization, once we get it, this will be merged and closed

**Comment by norberttech** [comment]
Closed by #571
